rules/sinks/storages/dynamodb/go.yaml

```diff
@@ -0,0 +1,10 @@
+sinks:
+
+  - id: Storages.AmazonDynamoDB.ReadAndWrite
+    name: Amazon Dynamo DB
+    domains:
+      - aws.amazon.com
+      - amazon.com
+    patterns:
+      - "(?i)(github.com/aws/aws-sdk-go/service/dynamodb).*(PutItem|GetItem|UpdateItem|DeleteItem)"
```

Comment:
Lets divide this into Read and ReadAndWrite

Reply:
done.